.github/workflows/r.yml

```diff
@@ -76,19 +86,15 @@ jobs:
           # Update this key to force a new cache
           prefix-key: "r-v1"
 
-      - name: Install build dependencies
-        run: |
-            R -e 'install.packages(c("nanoarrow", "geoarrow"))'
```

Comment:
It seems `install.packages()` doesn't work on Windows before setting CRAN mirror. So, just run `setup-r-dependencies` action instead.